sycl/test/basic_tests/interop-level-zero-2020.cpp

```diff
@@ -140,6 +146,8 @@ int main() {
                          bundle_state::executable>(
           {ZeKernelBundle.front(), ext::oneapi::level_zero::ownership::keep},
           Context);
+  kernel InteropKernel = make_kernel<backend::ext_oneapi_level_zero>(
+      {ZeKernel, ext::oneapi::level_zero::ownership::keep}, Context);
```

Comment:
Looks like it's missing kernel bundle as a first argument to constructor initializer list

Reply:
Right, fixed.